code/bit_manipulation/src/power_of_2/power_of_2.php

```diff
@@ -0,0 +1,17 @@
+<?php
+function is_Power($x)
+{
+
+
+ if (($x & ($x - 1)) != 0)
+		return "$x is not power of 2";
+ else
+    {
+		return "$x is power of 2";
+    }
+
+}
+
+print_r(is_Power(32)."\n");
+print_r(is_Power(31)."\n");
+?>
```

Comment:
Don't include a closing tag.

Reply:
OK Sir I have corrected it please review!